hub/src/server/authentication.ts

```diff
@@ -330,7 +379,17 @@ export class V1Authentication {
   }
 }
 
-export class LegacyAuthentication {
+export class LegacyAuthentication implements AuthenticationInterface {
+
+  checkAssociationToken(_token: string, _bearerAddress: string): void {
+    throw new Error('Method not implemented.')
+  }
+
+  parseAuthScopes(): AuthScopeValues {
+    // TODO: can legacy auth tokens be maliciously created to get around restrictive auth scopes?
```

Comment:
@kantai thoughts on this?

Reply:
No -- I don't think they'd be able to get around auth scopes any more than someone that possesses the private key already could.

However, we should look to deprecate support for these auth tokens -- blockstack.js hasn't used them for about a year now.